### PR TITLE
Make rewards tests sequental

### PR DESCRIPTION
**Pull request by Sriep**
Some other tests seemed to be interfering with token rewards tests, so changed to run tests sequentially.
Also, fix for https://github.com/0chain/0chain/pull/2220 adding new reward type.

Test with https://github.com/0chain/0chain/pull/2285 which might fix issue with missed block rewards.

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | miner_tests_sequental  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | miner_tests_sequental  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | piers_temp  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

File: tests/cli_tests/sharder_fee_rewards_test.go [hunk 1]
```diff
@@ -13,7 +13,8 @@ import (
 
 func TestSharderFeeRewards(testSetup *testing.T) { // nolint:gocyclo // team preference is to have codes all within test.
 	t := test.NewSystemTest(testSetup)
-	t.Skip("Skip till chain-side bugs are resolved")
+
+	t.Skip("wait for duplicate transaction issue to be solved, https://github.com/0chain/0chain/issues/2348")
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/sharder_fee_rewards_test.go:17]: Tracked in https://github.com/0chain/system_test/issues/658

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM

**Comment by Kishan-Dhakan** [comment]
Passing run https://github.com/0chain/system_test/actions/runs/4872994759/jobs/8691949215?pr=597

**Pull request merged**